ctru-rs/src/lib.rs

```diff
@@ -23,6 +23,7 @@
 #![feature(try_trait_v2)]
 #![feature(allocator_api)]
 #![feature(new_uninit)]
+#![feature(diagnostic_namespace)]
```

Comment:
This feature is stable, there is no need to add it to the list.

Reply:
Unfortunately it's not stable yet in the pinned compiler version we have... I could raise the `rust-version` in our crate to `1.78` and bump the pinned nightly, if you think that's preferable?

Reply:
Ah right. I'm for the idea of "if we use an unstable feature, we should update the MSRV as soon as it is stabilised" since stability is the biggest reason I see why we should bump the MSRV. 

Still, for such a small thing it isn't necessary. We can bump it later down the line whenever we see fit.